wrangler-api/src/main/java/io/cdap/wrangler/api/Directive.java

```diff
@@ -122,7 +123,7 @@
    *
    * @return a list of field operations about this directive
    */
-  default List<FieldTransformOperation> getFieldOperation() {
+  default List<FieldTransformOperation> getFieldOperations(StageContext context) {
```

Comment:
we should add this usage to the javadoc of the Directive interface

Reply:
I added a TODO for this method